### ISSUE TITLE
iOS11 iPhoneX - app cannot be shown in full screen

### ISSUE THREAD

**Issue by MarianXXL** [MvvmCross/MvvmCross-Samples#97]
My Xamarin.Forms app with Mvvmcross shows the app in the middle of the iPhoneX (simulator), not on the whole screen. It doesn't happen for a Xamarin.Forms app without Mvvmcross.

All MvvoCross sample apps have the same issu.

**Comment by MarianXXL** [comment]
![image](https://user-images.githubusercontent.com/30179352/30690950-38f08832-9ec6-11e7-9492-5162d7b8aa70.png)

**Comment by Cheesebaron** [comment]
Thanks for opening this issue. Please use and fill out the issue template as it gives us a better overview of what is going on. If you haven't already added reproduction steps or even better a repository with a reproduction sample, please do so. It speeds up debugging issues a lot.

**Comment by Cheesebaron** [comment]
Do you have 2x and 3x resource added to your project?

**Comment by MarianXXL** [comment]
Hi, thanks for your quick answer. The resources are OK, the app works on any othe device well. 
But I created a new Xamarin.Forms project and added MvvmCross.Forms and it works correct. It's a problem in my app. I see only some differeces to the new created app. I added in my Xamarin.Forms App the MvvmCross and in the new app I added MvvmCross.Forms. Therefore I use different derived classes. I must change it.
Thanks a lot,
warm regards
Marian
